test/distributed/tensor/test_random_ops.py

```diff
@@ -26,23 +26,22 @@
 from torch.distributed.tensor.parallel import ColwiseParallel, parallelize_module
 from torch.testing._internal.common_utils import run_tests
 from torch.testing._internal.distributed._tensor.common_dtensor import (
-    DTensorTestBase,
+    DTensorContinuousTestBase,
     skip_if_lt_x_gpu,
     skip_unless_torch_gpu,
-    with_comms,
 )
 
 
-class DistTensorRandomInitTest(DTensorTestBase):
+class DistTensorRandomInitTest(DTensorContinuousTestBase):
     def _run_init_op(self, init_op, *args, **kwargs):
-        device_mesh = self.build_device_mesh()
+        device_mesh = DeviceMesh(self.device_type(), torch.arange(self.world_size))
```

Comment:
is there a reason we change to `DeviceMesh()` instead of adding `build_device_mesh()`?

Reply:
we can add it, i was just hacking